src/screens/AddWalletScreen/ConfirmWalletDetails.tsx

```diff
@@ -186,20 +186,15 @@ function ConfirmWalletDetails({ route }) {
       <Box>
```

Comment:
The title of the modal is still Benefits of Taproot. But the content does not talk about it at all anymore. Need to replace the title or make the content about taproot